src/main/java/mekanism/client/gui/filter/GuiTItemStackFilter.java

```diff
@@ -128,7 +128,11 @@ protected void drawGuiContainerForegroundLayer(int mouseX, int mouseY) {
         fontRenderer.drawString(LangUtils.localize("gui.itemFilter.details") + ":", 35, 32, 0x00CD00);
         fontRenderer.drawString(LangUtils.localize("gui.itemFilter.min") + ":", 128, 20, 0x404040);
         fontRenderer.drawString(LangUtils.localize("gui.itemFilter.max") + ":", 128, 32, 0x404040);
-        fontRenderer.drawString(LangUtils.localize("gui." + (filter.sizeMode ? "on" : "off")), 141, 46, 0x404040);
+        String sizeModeString = LangUtils.localize("gui." + (filter.sizeMode ? "on" : "off"));
+        if (tileEntity.singleItem && filter.sizeMode)
```

Comment:
Please always use braces.

If you're using IntelliJ IDEA, this can be imported (save it to the project, no overwrite your defaults): https://github.com/mekanism/Mekanism/blob/1.12/docs/GoogleStyle4Indents.xml

Reply:
I have it imported, yet it showed me nothing wrong with missing braces, could you maybe point me in some direction where i can look to fix this?
![image](https://user-images.githubusercontent.com/1835434/57552008-ccba4d80-736a-11e9-8672-99796e092c91.png)

Reply:
On the left under `Code Style` click on the sub tab for `Java`. It should have things like `Force braces` set to `always` under the `Wrapping and Braces` tab for if statements. My guess is that for some reason the scheme didn't load for the java specific scheme.

Reply:
strange, I do have it turned on. So probably some cache problem at my side, will try clearing it.
![image](https://user-images.githubusercontent.com/1835434/57552935-b9f54800-736d-11e9-97e9-6dd719b233b6.png)
thanks anyway

Reply:
Actually thinking about it, probably has to do with it not automatically running. (As in if you didn't put the braces out of habit anyways it wouldn't add them until you run the formatter `CTRL+ALT+L` for current file. For now if you want you can just add the braces manually though given after we get the open PRs merged I am going to do a quick pass making sure things are easier to read across the board since we are changing some of the formatting styling in #5463 .